QuickOSM/core/parser/osm_parser.py

```diff
@@ -122,6 +122,29 @@ def processing_parse(self):
                 message = 'Error on the layer : {layer}'.format(layer=layer)
                 raise QuickOsmException(message)
 
+            if self.feedback:
+                self.feedback.pushInfo('Checking the validity of the geometry of the layer {}.'.format(layer))
+            validity = processing.run(
+                "qgis:checkvalidity", {
+                    'INPUT_LAYER': layers[layer]['vectorLayer'],
+                    'METHOD': 2,
```

Comment:
```suggestion
                    'METHOD': 2,  # What is number 2 ?
```

Easier to read instead of going to check in QGIS what is this enum and this value for.